tools/NewModuleScript/MainScreenDialogTemplate.kt

```diff
@@ -0,0 +1,129 @@
+/* Copyright 2023 Esri
```

Comment:
Nitpick:
```suggestion
/* Copyright 2025 Esri
```

Reply:
This template will be used and updated by the NewModuleScript [here](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/blob/shubham/quality-enhancements/tools/NewModuleScript/src/main/kotlin/NewModuleScript.kt#L276)